php/pantheon/checks/database.php

```diff
@@ -44,7 +44,7 @@ protected function countRows() {
 		global $wpdb;
 		foreach ( $this->getTables() as $table ) {
 			$this->tables[$table->TABLE_NAME] = $table;
-			if ( preg_match( "#.*_options#s", $table->TABLE_NAME ) ) { 
+			if ( $table->TABLE_NAME == $wpdb->prefix . 'options' ) { 
```

Comment:
Can/should this be `===` instead of `==`? IMO strict checking should always be used unless there's a specific reason to not.